MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 recursive-include wagtail_content_import/templates *
 recursive-include wagtail_content_import/pickers/google/templates *
-recursive-include wagtail_content_import/pickers/google/static *
+recursive-include wagtail_content_import/pickers/google/static*
```

Comment:
The asterisk seems to have moved here, not sure if this matters but it no longer matches the previous lines